index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Advisory Mentorship</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+
+    <meta name="description" content="The Advisory Mentorship Program" />
+    <!-- Google -->
+    <meta itemprop="name" content="The Advisory Mentorship Program" />
+    <meta itemprop="description" content="The Advisory Mentorship Program" />
```

Comment:
"Program" should be "Programme". Also, could I check where the meta tags originally came from?

Reply:
I think its from [here](https://github.com/AdvisorySG/mentorship-page/blob/3090e2fb7d4d1447d6b4d40d463ee9975dd77195/public/index.html)

Reply:
That explains a lot, I haven't touched that file in so long that I forgot it existed